_field-types/supported-field-types/wildcard.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: Wildcard
+nav_order: 62
+has_children: false
+parent: String field types
+grand_parent: Supported field types
+---
+
+# Wildcard field type
+
+A `wildcard` field is a variant of a `keyword` field designed for arbitrary substring and regular expression matching.
+
+Use a `wildcard` field when your content consists of "strings of characters" and not "text". Examples include unstructured log lines and computer code.
+
+The `wildcard` field type is indexed differently from the `keyword` field type. Whereas `keyword` fields write the original field value to the index, the `wildcard` field type splits the field value into substrings with a length that is less than or equal to 3 and writes the substrings to the index. For example, the string `test` is split into strings `t`, `te`, `tes`, `e`, `es`, and `est`. 
+
+At search time, required substrings from the query pattern are matched against the index to produce candidate documents, which are then filtered according to the pattern in the query. For example, for the search term `test`, OpenSearch performs an indexed search for `tes AND est`. If the search term contains less than three characters, OpenSearch uses character substrings that are one or two characters long. For each matching document, if the source value is `test`, then the document is returned in the results. This excludes false positive values like `nikola tesla felt alternating current was best`.
+
+In general, exact match queries (like [`term`]({{site.url}}{{site.baseurl}}/query-dsl/term/term/) or [`terms`]({{site.url}}{{site.baseurl}}/query-dsl/term/term/) queries) perform less effectively on `wildcard` fields than on `keyword` fields, while [`wildcard`]({{site.url}}{{site.baseurl}}/query-dsl/term/wildcard/), [`prefix`]({{site.url}}{{site.baseurl}}/query-dsl/term/prefix/), and [`regexp`]({{site.url}}{{site.baseurl}}/query-dsl/term/regexp/) queries perform better on `wildcard` fields.
+{: .tip}
+
+## Example
+
+Create a mapping with a `wildcard` field:
+
+```json
+PUT movies
```

Comment:
Oh... I changed the field name from the example that I copied to highlight that wildcard fields are good for log lines, but I forgot to change the name of the index.

Reply:
@msfroh Changed the index name to "logs"